lms/services/h_api.py

```diff
@@ -114,16 +111,18 @@ def do_upsert_group():
             self.create_user(HUser("lms", provider="lms", provider_unique_id="lms"))
             do_upsert_group()
 
-    def add_user_to_group(self, h_user, group_id):
+    def add_user_to_group(self, h_user, h_group):
         """
         Add the user as a member of the group.
 
         :param h_user: the user to add to the group
         :type h_user: HUser
-        :param group_id: The id of the group
+        :param h_group: the group to add the user to
+        :type h_group: HGroup
         """
         self._api_request(
-            "POST", f"groups/{group_id}/members/{h_user.userid(self._authority)}"
+            "POST",
+            f"groups/{h_group.groupid(self._authority)}/members/{h_user.userid(self._authority)}",
```

Comment:
This looks outwardly daft as we just pluck out things we could be given, but the interface is really clean as a result. Shame we're about to delete it :).